pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -351,3 +352,95 @@ func Write(ctx context.Context, dbInfo *DBInfo, store string, deletes storage.De
 
 	return nil
 }
+
+func WriteAuthorizationModel(ctx context.Context, dbInfo *DBInfo, store string, model *openfgav1.AuthorizationModel) error {
```

Comment:
Seems DBInfo is overkill for this method given we only need the statement builder out of it, and we could probably do a pass over `sqlcommon` to improve things a bit, but for consistency with the other signatures for now we can move forward.